src/message/formatter-runtime-bind.js

```diff
@@ -0,0 +1,45 @@
+define(function() {
+
+return function( cldr, messageformatter ) {
+	var locale = cldr.locale,
+		origToString = messageformatter.toString;
+
+	messageformatter.toString = function() {
+		var argNames, argValues, output,
+			args = {};
+
+		output = origToString.call( messageformatter );
+
+		if ( /number\(/.test( output ) ) {
+			args.number = "messageFormat.number";
+		}
+
+		if ( /plural\(/.test( output ) ) {
+			args.plural = "messageFormat.plural";
+		}
+
+		if ( /select\(/.test( output ) ) {
+			args.select = "messageFormat.select";
+		}
+
+		output.replace( /pluralFuncs(\[([^\]]+)\]|\.([a-zA-Z]+))/, function( match ) {
```

Comment:
I have no idea what this is doing. This specifically, the whole file in general.